src/wavesurfer.js

```diff
@@ -910,16 +910,22 @@ export default class WaveSurfer extends util.Observer {
 
     /**
      * Horizontally zooms the waveform in and out. It also changes the parameter
-     * `minPxPerSec` and enables the `scrollParent` option.
+     * `minPxPerSec` and enables the `scrollParent` option. Calling the function
+     * with a falsey parameter will reset the zoom state.
      *
-     * @param {number} pxPerSec Number of horizontal pixels per second of audio
+     * @param {number?} pxPerSec Number of horizontal pixels per second of
```

Comment:
By the way, a nullable type is _prefixed_ with a `?`, I think (according to http://usejsdoc.org/tags-type.html).

Reply:
aah true. I'll correct and merge.